modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue

```diff
@@ -20,12 +20,30 @@
         :icon-only="true" icon="delete-icon"
         type="outline"
       /> -->
-      <AposContextMenu
-        v-if="more.menu.length"
-        :button="more.button"
-        :menu="more.menu"
-        @item-clicked="batchAction"
-      />
+      <div
+        v-for="{name, label, icon, operations} in showedOperations"
+        :key="name"
+      >
+        <AposButton
+          v-if="!operations"
+          label="label"
+          :icon-only="true"
+          :icon="icon"
+          type="outline"
+          @click="() => {}"
```

Comment:
this should have the same click handling as the context menu. So run batchAction with the action.

Reply:
Modified, but has been modified in the next PR.